force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -241,8 +314,11 @@ public with sharing class FlowControlLoopsCollections {
         Set<String> stringSet = new Set<String>();
 
         //add strings to the set
+        for ( String uniqueString : inputList){
+            stringSet.add(uniqueString);
+        }
 
-        return null; // Replace null with the variable you used to store the result
+        return stringSet; // Replace null with the variable you used to store the result
```

Comment:
good job. You can also pass a List directly to the Set constructor and skill all the iteration. 

 public static Set<String> createStringSet(List<String> inputList) {
        return new Set<String>(inputList); // Replace null with the variable you used to store the result
    }